modules/rpm-ostree/README.md

```diff
@@ -0,0 +1,22 @@
+# [`rpm-ostree`](https://coreos.github.io/rpm-ostree/) module for startingpoint
+
+The `rpm-ostree` module offers pseudo-declarative package and repository management using `rpm-ostree`.
+
+The module first downloads the repository files from repositories declared under `repos:` into `/etc/yum.repos.d/`. The magic string `%OS_VERSION%` is substituted with the current VERSION_ID (major Fedora version), which can be used, for example, for pulling correct versions of repositories from [Fedora's Copr](https://copr.fedorainfracloud.org/).
+
+Then the module installs the packages declared under `install:` using `rpm-ostree install`, and lastly, it removes the packages declared under `remove:` using `rpm-ostree override remove`.
+
+Unfortunately, currently `rpm-ostree override remove`, and this module, might not be able to remove packages installed in image builds. Packages included by Fedora, such as Firefox can still be removed, though.
```

Comment:
Is this accurate? Every time I wonder this out loud I get a bit of a different answer. Worth including in the docs, even? What about `remove` being like `git rm` in that it doesn't reclaim disc space?

Reply:
I think it is able to remove packages installed in image builds iirc, I'll need to check with that but i'm pretty positive that is an option